src/views/SendConfirm.js

```diff
@@ -117,8 +118,27 @@ const SendConfirm = () => {
 
     setIsLoadingConfirmationShown(true);
 
+    function additionalAction() {
+      dispatch(
+        notify({
+          text: translate('SendConfirm.errorMessageTransferUnknown'),
+          type: NotificationsTypes.ERROR,
+        }),
+      );
+      setIsLoadingConfirmationShown(false);
+    }
+
     try {
-      await dispatch(transfer(address, amount, paymentNote));
+      await dispatch(
+        transfer(
+          address,
+          amount,
+          paymentNote,
+          PATHFINDER_HOPS_DEFAULT,
+          PATHFINDER_HOPS_DEFAULT + 1,
+          additionalAction,
```

Comment:
the idea behind this is it gives us flexibility what to do in case of for example endless spinner. We define that additional action where transfer function gets dispatched. Our loading spinners have state in various components so thus that should be closed from there